### ISSUE TITLE
novinky.cz: ads

### ISSUE THREAD

**Issue by llair** [uBlockOrigin/uAssets#17255]
### Prerequisites

- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I forced an update of my filter lists. (Click the ["Purge all caches"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches) button while holding the 'Shift' key, then click the ["Update now"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now) button.)
- [X] I did not remove any of the [default filter lists](https://user-images.githubusercontent.com/585534/91296476-ab9d8c00-e76a-11ea-9fb9-87d6861bdf73.png), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable external filter lists, or I have verified that the issue still occurs without enabling external filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin (uBO) along with other content blocker extensions.
- [X] I have verified that the web browser's built-in blocker or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through a VPN) is not causing the issue.
- [ ] I did not answer truthfully to **ALL** the above checkpoints.

### URL address of the web page

`https://www.novinky.cz/`

### Category

ads

### Description

All ads are shown on the webpage. Clearing cookies and filter cache doesn't help.

![novinky](https://user-images.githubusercontent.com/76915553/226839953-742469c3-8b23-4c56-89e1-35e965dc5a80.png)

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.47.4
Chromium: 111
filterset (summary): 
  network: 86925
  cosmetic: 39532
  scriptlet: 17187
  html: 0
listset (total-discarded, last-updated): 
  default: 
    user-filters: 0-0, never
    CZE-0: 736-0, now
    easylist: 64293-46, now
    easyprivacy: 31065-1177, now
    plowe-0: 3752-931, now
    ublock-abuse: 78-0, now
    ublock-badware: 4748-114, now
    ublock-filters: 32031-83, now
    ublock-privacy: 293-1, now
    ublock-quick-fixes: 421-7, now
    ublock-unbreak: 1936-39, now
    urlhaus-1: 6859-0, now
filterset (user): [empty]
trustedset: 
  added: [array of 24 redacted]
switchRuleset: 
  added: [array of 1 redacted]
modifiedUserSettings: [none]
modifiedHiddenSettings: [none]
supportStats: 
  allReadyAfter: 1983 ms (selfie)
  maxAssetCacheWait: 259 ms
popupPanel: 
  blocked: 30
  blockedDetails: 
    novinky.cz: 3
    adform.net: 1
    adition.com: 1
    adnxs.com: 2
    adsrvr.org: 1
    amazon-adsystem.com: 1
    audrte.com: 1
    bidr.io: 1
    bidswitch.net: 1
    criteo.com: 1
    crwdcntrl.net: 1
    de17a.com: 1
    dotomi.com: 1
    doubleclick.net: 3
    everesttech.net: 1
    gemius.pl: 1
    googletagmanager.com: 1
    mathtag.com: 1
    quantserve.com: 1
    seznam.cz: 1
    simpli.fi: 1
    sitescout.com: 1
    stackadapt.com: 1
    weborama.fr: 1
    yahoo.com: 1
```
</details>

**Comment by krystian3w** [comment]
Recommended is copy trouble to already closed topic instead open new as long random of user blocked you:

https://github.com/uBlockOrigin/uAssets/issues?q=is%3Aissue+sort%3Aupdated-desc+%22novinky.cz%22+in%3Atitle+

**Comment by garry-ut99** [comment]
>  * [x]  I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)

No, a duplicate of: https://github.com/uBlockOrigin/uAssets/issues?q=is%3Aissue+in%3Atitle+%22novinky.cz%3A+ads%22+

**Comment by llair** [comment]
> @llair update all your lists now:
> 
> (Click the ["Purge all caches"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches) button while holding the 'Shift' key, then click the ["Update now"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now) button.)

I've tried this several times for the last week. Today, it worked. Let's hope it stays that way. Thanks
